quarkus/service/src/test/java/org/apache/polaris/service/quarkus/catalog/BasePolarisCatalogTest.java

```diff
@@ -1458,6 +1461,55 @@ public void testDropTableWithPurgeDisabled() {
         .hasMessageContaining(PolarisConfiguration.DROP_WITH_PURGE_ENABLED.key);
   }
 
+  @Test
+  public void testRefreshIOForTableLike() {
+    // Enable ALLOW_SPECIFYING_FILE_IO_IMPL and disable SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION
+    PolarisConfigurationStore configurationStore =
+        new DefaultConfigurationStore(
+            Map.of(
+                "ALLOW_SPECIFYING_FILE_IO_IMPL",
+                true,
+                PolarisConfiguration.SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION.key,
+                false));
+    try (MockedStatic<CatalogUtil> catalogUtil =
+        Mockito.mockStatic(CatalogUtil.class, Mockito.CALLS_REAL_METHODS)) {
```

Comment:
Oh, please do not mock anything static. I know this is not powermock, but mocking static stuff requires some "interesting" instrumentation. You wrap the call to CatalogUtil from DefaultFileIOFactory in a separate method and override that method here. Something like:

```java
class DefaultFileIOFactory {
   @VisibleForTesting
   FileIO callCatalogUtil(params) {
     return CatalogUtil.load...
   }
}
```
and here 
```java
new DefaultFileIOFactory(params) {
   @Override 
   FileIO callCatalogUtil(params) {
     // run your assertions
     return super.callCatalogUtil(...)   // or not :shrug:
}
```

Reply:
Done, create a separate class for the FileIOFactory.